portfolio/src/main/java/com/google/sps/data/Entry.java

```diff
@@ -12,9 +12,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/* Class for specifying comment text and metadata. */
```

Comment:
Move the comment to be just above the class definition

Reply:
Done